### ISSUE TITLE
Wrong video frame loaded when scrolling (need to wait a little for the correct video to load)

### ISSUE THREAD

**Issue by ShravanMeena** [Shopify/flash-list#847]
## Current behavior
I have a flashlist of View elements, which display information about the products (name). While scrolling the wrong videos of the items are shown. Only after waiting a little bit, the correct video replace the wrong ones, and everything looks as it's supposed to look.

```I shared behaviour video here```

https://github.com/Shopify/flash-list/assets/41790742/742254b2-9453-430b-9f8d-23d38b44117a

## Expected behavior
Scrolling should not affect how video are displayed: each corresponding video should be displayed correctly.

## To Reproduce
https://github.com/ShravanMeena/flashlist-POC

## Platform:
- [ ] Android

## Environment
 "@shopify/flash-list": "^1.4.3",
 
 Need Help 😭

**Comment by oguzydz** [comment]
same with photo

**Comment by micasabyKanika** [comment]
Anyone find solution?

**Comment by swikars1** [comment]
This because of the internal implementation of the flashlist.
I fixed it by using thumbnailUrl as key prop in the parent of the render item. Does the job foe me but obviously you'll loose flashlist's performance.

**Comment by ShravanMeena** [comment]
> This is because of the internal implementation of the flashlist. I fixed it by using thumbnailUrl as key prop in the parent of the render item. Does the job foe me but obviously you'll loose flashlist's performance.

I have already did same.but after adding key to parents loose recycling or performance

**Comment by timotismjntk** [comment]
Yeah it's expected behavior, because it's recycled the previous cell

**Comment by pacehutt** [comment]
that's literally causing performance issues and creating bad user experience

**Comment by naqvitalha** [comment]
Hi there! 👋 

Thank you for reporting this issue. We appreciate your contribution to making FlashList better!

We wanted to let you know that we're no longer actively supporting FlashList v1, as our focus has shifted to v2 which includes significant improvements and bug fixes.

If you're still experiencing this issue, we'd encourage you to upgrade to FlashList v2. If the problem persists after upgrading, please feel free to open a new issue and we'll be happy to help!

You can find the migration guide and v2 documentation at: https://shopify.github.io/flash-list/

Thanks again for your understanding and for using FlashList! 🚀
